Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                var surname = Lesson4.PromptString("Please enter your surname: ");
+                var age = Lesson4.PromptInt("Please enter your age: ");
+                var weight = Lesson4.PromptFloat("Please enter your weight (in kg): ");
+                var height = Lesson4.PromptFloat("Please enter your height (in m): ");
+                Console.WriteLine(surname + " is " + age + " " + "years old, his weight is " + weight + "kg and his height is " + height + "m.");
+
+                var Bmi = Lesson4.CalculateBmi(weight, height);
+                Console.WriteLine("Your estimated BMI is " + Bmi);
+            }
+        }
+
+        public static string PromptString(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+            var userName = validateString(input);
+            if (userName == "-")
+            {
+                Console.WriteLine("Name cannot be empty.");
+            }
+            return userName;
+        }
+
+        public static string validateString(string input)
+        {
+            if (String.IsNullOrEmpty(input))
+            {
+                return "-";
+            }
+            return input;
+        }
+
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+            if (String.IsNullOrEmpty(input)) return 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886678)